recipe/meta.yaml

```diff
@@ -19,6 +19,8 @@ build:
 
 requirements:
   build:
+    - krb5 1.14.*
+    - libssh2 1.8.*
     # python is a build requirement on Windows to resolve features.
     - python           # [win]
     # perl is required to run the tests on UNIX.
```

Comment:
What's the exact reason behind pinning to the latest version? Is the reason documented somewhere?